src/views/results/index.ts

```diff
@@ -480,6 +493,53 @@ async function runHandler(options?: StatementInfo) {
   }
 }
 
+async function statementToRpgDs(statement: ParsedStatementInfo) : Promise<string> {
+  setCancelButtonVisibility(true);
+  updateStatusBar({executing: true});
+  const result = await JobManager.runSQLVerbose(statement.content, undefined, 1);
+  setCancelButtonVisibility(false);
```

Comment:
Sorry to be picky. I don't think this should be in this function. This function should purely generate code and nothing else, and therefore the parameter to the function should be the result of `runSQLVerbose`.